README.md

```diff
@@ -187,11 +187,7 @@ value = Setting.get("SETTING_NAME", default="django-extra-settings")
 ```python
 from extra_settings.models import Setting
 
-setting_obj = Setting(
-    name="SETTING_NAME",
-    value_type=Setting.TYPE_BOOL,
-    value=True,
-)
+setting_obj, created = Setting.objects.get_or_create(name="SETTING_NAME")
```

Comment:
There is no reason for changing the example setting name.

Reply:
Can you show me how to update a setting value without changing this?   I can not get the example in the readme you provided to work.